build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM docker.io/openshift/origin-release:golang-1.15 as build
+FROM registry.svc.ci.openshift.org/origin/builder:golang-1.15 as build
```

Comment:
Is there any difference between this and `registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7` image ?

Reply:
I'm not familiar with the other image, this is the equivalent of the previous image

Reply:
Its the image used in [WMCB](https://github.com/openshift/windows-machine-config-bootstrapper/blob/master/Dockerfile.tools#L1)